packages/web-components/src/_docs/getting-started/overview.stories.mdx

```diff
@@ -2,19 +2,21 @@ import { Meta } from '@storybook/addon-docs';
 
 <Meta title="Getting Started/Overview" />
 
-## Overview
+## The official Fluent UI Web Component library
 
-## What are the Fluent UI Web Components?
+###
 
-The Fluent UI Web Components is a library of Web Components that composes `@microsoft/fast-foundation`, styled with design tokens for Microsoft's [Fluent design language](https://github.com/microsoft/fluentui).
-It is designed to get you up and running quickly with an extensible set of Web Components for building Fluent UI web applications.
+### Web Component libraries for building Fluent UI web experiences
```

Comment:
Considering this is specific to one library, is there a better "heading" for this section of text?

Reply:
You're right. This is unclear and the whole page reads better w/o this subhead. Removed